packages/calypso-e2e/src/element-helper.ts

```diff
@@ -31,3 +38,38 @@ export async function waitForElementEnabled(
 		page.waitForFunction( ( element: any ) => element.ariaDisabled !== 'true', elementHandle ),
 	] );
 }
+
+/**
+ * Locates and clicks on a specified tab on the NavTab.
+ *
+ * NavTabs are used throughout calypso to contain multiple related but separate pages within the same
+ * overall page. For instance, on the Media gallery page a NavTab is used to filter the gallery to
+ * show a specific category of gallery items.
+ *
+ * @param {Page} page Underlying page on which interactions take place.
+ * @param {string} name Name of the tab to be clicked.
+ */
+export async function clickNavTab( page: Page, name: string ): Promise< void > {
```

Comment:
I realize in the past my decision for the project was to discourage the creation of helper methods. I still feel that way for the most part, but now have shifted my stance to allow for more exceptions especially when similar method is used across quite a few files.